torch/_inductor/lowering.py

```diff
@@ -4996,6 +4996,27 @@ def cumprod(x, axis=None, dtype=None):
     return result
 
 
+@register_lowering(aten.logcumsumexp)
+def logcumsumexp(x, dim):
+    def log_add_exp_helper(a, b):
+        min_v = ops.minimum(a, b)
+        max_v = ops.maximum(a, b)
+        if min_v != max_v or ops.isfinite(min):
```

Comment:
Use `ops.where` and `ops.*` handlers everywhre (ne, ops.or...)

Reply:
I did catch this with some help from peter. Interesting to me that the `!=` is not handled by operator overloading the way `+` and `-` are on the next line for example. Is there a reason that logical operators are not overloaded in this way or is that an oversight that could be corrected?

Reply:
btw here is where the operators are implemented:
https://github.com/pytorch/pytorch/blob/b73a2b779586b23e62759cc41b3b0724bbfa3080/torch/_inductor/virtualized.py#L802-L803